Sources/OpenId4VP/OpenId4VP.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+public class OpenId4VP {
+    let traceabilityId: String
+    let networkManager: NetworkManaging
+    var authorizationRequest: AuthorizationRequest?
+    static var presentationDefinitionId: String?
+    static var responseUri: String?
+    
+    init(traceabilityId: String, networkManager: NetworkManaging? = nil) {
+        self.traceabilityId = traceabilityId
+        self.networkManager = networkManager ?? NetworkManager.shared
+    }
+    
+    public static func setPresentationDefinitionId(_ id: String) {
+        OpenId4VP.presentationDefinitionId = id
+    }
+    
+    public static func setResponseUri(_ responseUri: String){
+        OpenId4VP.responseUri = responseUri
```

Comment:
same comment as of setPresentationDefinitionId function

Reply:
refactored it.